test/functional/filters/apiaudit/api_audit_filter_test.go

```diff
@@ -0,0 +1,91 @@
+package apiaudit
```

Comment:
do we need to restrict this to vector only?

Reply:
Yes